src/components/Modals/CreateGnosisSafeModal/index.tsx

```diff
@@ -65,12 +67,15 @@ const CreateGnosisSafeModalContent: FunctionComponent<{
 				await addDAO(gnosisAddress)
 				await editDAO({gnosisAddress, name: daoName})
 				toastSuccess("DAO successfully created!")
+				setIsOpened(false)
 				afterCreate()
 			} catch (e) {
 				console.error(e)
 				toastError("Failed to create DAO")
 			}
 			setProcessing(false)
+		} else if (stage === "import") {
+			toastWarning("This feature is not yet supported. Please, try later.")
```

Comment:
This is good for user interaction, but why i like console logs is that they trigger eslint modals and help not to forget about it.
Please, revert to console logs for now, and I suggest to turn all TODO console logs to alerts in one PR, so we can keep both eslint warnings and notify user

Reply:
What about using something like https://www.npmjs.com/package/leasot ?
Basically, it parses TODO/FIXME comments and displays them to you.
I'm getting an idea to be "warned" about TODOs that we have, but we would always have something not implemented yet, and always having linter complaining about the console (which might not necessarily be TODO but some debugging leftover) is quite annoying. WDYT about removing all consoles, adding user messages, and introducing `leasot` (or any other similar tool)? I can also poke around injecting that into console output for dev server, so it would proactively remind us about todos

Reply:
> What about using something like https://www.npmjs.com/package/leasot ? Basically, it parses TODO/FIXME comments and displays them to you. I'm getting an idea to be "warned" about TODOs that we have, but we would always have something not implemented yet, and always having linter complaining about the console (which might not necessarily be TODO but some debugging leftover) is quite annoying. WDYT about removing all consoles, adding user messages, and introducing `leasot` (or any other similar tool)? I can also poke around injecting that into console output for dev server, so it would proactively remind us about todos

That's just proposal - I added console for now anyway, just FYI

Reply:
I don't like the idea of using external tools for such a minor thing.

Reply:
It's not really a tool, but just a library that clearly dev dependency. So pretty much like prettier or eslint, but for handling different sorts of code quality. But I'm not pushing for that, will be fine with not using that (haven't used that previously anyway, just something I quickly found)